lib/sidekiq-status/web.rb

```diff
@@ -75,8 +75,11 @@ def has_sort_by?(value)
 
       app.get '/statuses' do
 
-        namespace_jids = Sidekiq.redis{ |conn| conn.keys('sidekiq:status:*') }
-        jids = namespace_jids.map{ |id_namespace| id_namespace.split(':').last }
+        jids = Sidekiq.redis do |conn|
+          conn.scan_each(match: 'sidekiq:status:*', count: 100).map do |key|
+            key.split(':').last
+          end
+        end
```

Comment:
Call `#uniq` to remove duplicate elements before as SCAN may return the same key multiple times per the Redis docs.

Reply:
Good find! I actually did not know that about SCAN, I always assumed it maintained a state.